stable_baselines3/common/vec_env/vec_frame_stack.py

```diff
@@ -13,15 +13,22 @@ class VecFrameStack(VecEnvWrapper):
 
     :param venv: the vectorized environment to wrap
     :param n_stack: Number of frames to stack
+    :param dim: Dimension along which to stack frames (not including batch dimension).
```

Comment:
Could mention here it also supports the negative-dimensions as slicing/indexing usually does.